documentation/API-design-guidelines.md

```diff
@@ -1976,40 +1976,46 @@ response:
 204 No Content
 ```
 
-## Appendix A: `info.description` template for `device` identification from access token
+## Appendix A (Normative): `info.description` template for when API subject identification can be from either an access token or explicit identifier
 
-The documentation template below is recommended to be used as part of the API documentation in `info.description` property in the CAMARA API specs which use the `device`object defined in [CAMARA_common.yaml](/artifacts/CAMARA_common.yaml) artifact. This template provides guidance on how to handle device information in API requests **when using 3-legged access tokens and the device can be uniquely identified by the token**.
+When an API requires a User or Resource Owner (the "subject" of the API) to be identified in order to get access to the subject's data, that subject can be identified in one of two ways:
```

Comment:
I would not write "one or two ways" because operator token is a third way.

I think that a valid access token always contains or is associated with the information the API implementation needs. How that information is obtained is not something that should be written in each API description.

Reply:
The "two ways" are:
- 3-legged access token
- explicit identifier in the API service call

What other ways are there? `operatortoken` is already included in the first way.